jetty9-base/src/main/jetty-config/jetty.commands

```diff
@@ -1,6 +1,6 @@
 --create-startd
 
---add-to-start=server,webapp,http,deploy,jsp,jstl,http-forwarded,resources,gae
```

Comment:
I like the idea of a gcp module and a `gcp.commands` file that will apply to the `*.ini` Jetty config files.